test/mocks/fake.go

```diff
@@ -59,9 +62,10 @@ func (f *Fake) AddMockedUnstructuredServiceBindingRequest(
 	backingServiceResourceRef string,
 	applicationResourceRef string,
 	matchLabels map[string]string,
+	groupVersionResource schema.GroupVersionResource,
```

Comment:
And the same for the other occurrences :-)

Reply:
Well, all this gvr thing is resolved by somebody's PR.